From 9ebdb8404be0deb4fddc768d66a72bb0d8f7cc28 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Sun, 19 May 2024 13:34:50 +0300
Subject: [PATCH 079/155] compulab-imx8m-plus: add support for machine specific
 environment add-on

Add support for machine specific environment add-on which should be defined
in MACHINE_EXTRA_ENV_SETTINGS macro prior to including "compulab-imx8m-plus.h"

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 include/configs/compulab-imx8m-plus.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/configs/compulab-imx8m-plus.h b/include/configs/compulab-imx8m-plus.h
index 9efc896f538..979ba126c93 100644
--- a/include/configs/compulab-imx8m-plus.h
+++ b/include/configs/compulab-imx8m-plus.h
@@ -27,6 +27,10 @@
 #define BOOTENV
 #endif
 
+#ifndef MACHINE_EXTRA_ENV_SETTINGS
+#define MACHINE_EXTRA_ENV_SETTINGS
+#endif
+
 #define CFG_MFG_ENV_SETTINGS \
 	CFG_MFG_ENV_SETTINGS_DEFAULT \
 	"initrd_addr=0x43800000\0" \
@@ -38,6 +42,7 @@
 #define CFG_EXTRA_ENV_SETTINGS		\
 	CFG_MFG_ENV_SETTINGS \
 	BOOTENV \
+	MACHINE_EXTRA_ENV_SETTINGS \
 	"video_link=1\0" \
 	"stdout=serial,vidconsole\0" \
 	"stderr=serial,vidconsole\0" \
-- 
2.34.1

